services/conference/src/scripts/components/map/Base/Base.tsx

```diff
@@ -157,6 +160,16 @@ export const Base: React.FC<BaseProps> = (props: BaseProps) => {
     },
     [outer],
   )
+  // scroll range
+  useEffect(
+    () => {
+      const orgMat = new DOMMatrix(matrix.toString())
+      setMatrix(matrix.translate(MAP_SIZE /2, MAP_SIZE /2))
+      setMatrix(matrix.translate(-MAP_SIZE /2, -MAP_SIZE /2))
```

Comment:
Sorry I don't understand the purpose of this effect, could you explain a bit?
It seems finally the matrix is not changed.

Reply:
Thank you for reivew.
Sorry this is not my intention. It will be made by merge and I've overlooked it.

Reply:
I've removed two lines.